stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/controller/ChannelControllerImpl.kt

```diff
@@ -87,7 +88,7 @@ internal class ChannelControllerImpl(
 ) :
     ChannelController {
     private val editJobs = mutableMapOf<String, Job>()
-    private val _messages = MutableLiveData<Map<String, Message>>()
+    private val _messages = MutableLiveData<Pair<Boolean, Map<String, Message>>>()
```

Comment:
Honestly, I think it's not right to create for the source of live data for messages of a channel such data structure as Pair<Boolean, Map<String, Message>>. Seems too complicated, for such a case I'd like to have some data class:
``` kotlin
 data class MessagesStates(val messages: List<Message>, val loadingState: Booleand = false)
```
or even clearer:
``` kotlin
sealed class MessagesState {
 class IdleMessages(val messages: List<Message>) : MessagesState()
 class LoadingMoreMessages(val messages: List<Message>) : MessagesState()
}
```

Reply:
Ok. I'll do that.